### PR TITLE
feat: PE/MSE Implementation for WebTorrent - update for new version

### PR DESCRIPTION
**Pull request by ivan1986**
Update PR #36 for new version

### CODE REVIEW

**Comment by kelvin-lu** [comment]
@ivan1986 just wanted to say thanks and you have my eternal gratitude for updating my stale PR. Apologies for never following through.

**Comment by feross** [comment]
@ivan1986 There are still conflicts in this PR -- do you mind updating it once more?

@webtorrent/team -- is there interest from anyone on the WebTorrent core team to shepherd this PR and do a detailed code review, checking that it complies with the spec, is secure, and works with other clients?

**Comment by ivan1986** [comment]
@feross updated, fix conflicts

also
> works with other clients

checked popcorn time vs deluge - works fine with forse option and other of couse

**Comment by DiegoRBaquero** [comment]
A simple `standard --fix` or use of `lebab` could update the `var`s, I can review once tests pass

File: index.js [hunk 1]
```diff
@@ -177,6 +217,60 @@ class Wire extends stream.Duplex {
     this._push(MESSAGE_KEEP_ALIVE)
   }
 
+  sendPe1 () {
```
Comment (DiegoRBaquero) [inline index.js:220]: Why are these functions never used?

Would be great to have at least minimum tests

Comment (ivan1986) [reply index.js:220]: https://github.com/webtorrent/webtorrent/pull/1820/files#diff-e5eeda71546bb4a993c32e575d0c54c2ef19adffc968e3f63cf20e623bcedb52R208

**Review by DiegoRBaquero — APPROVED** [review]

**Comment by h1z1** [comment]
Given this was merged can you clarify where that prime came from?

> const DH_PRIME = 'ffffffffffffffffc90fdaa22168c234c4c6628b80dc1cd129024e088a67cc74020bbea63b139b22514a08798e3404ddef9519b3cd3a431b302b0a6df25f14374fe1356d6d51c245e485b576625e7ec6f44c42e9a63a36210000000000090563'

**Pull request merged**